tests/FileManipulation/FileManipulationTest.php

```diff
@@ -30,10 +30,11 @@ public function setUp()
      * @param string $php_version
      * @param string[] $issues_to_fix
      * @param bool $safe_types
+     * @param bool $allow_backwards_incompatible_changes
      *
      * @return void
      */
-    public function testValidCode($input_code, $output_code, $php_version, array $issues_to_fix, $safe_types)
+    public function testValidCode($input_code, $output_code, $php_version, array $issues_to_fix, $safe_types, bool $allow_backwards_incompatible_changes)
```

Comment:
you could add a default, and then you wouldn't need to update every test (I don't think)